### DISCUSSION TITLE
Changing the shortcut to open the window of copyq

### DISCUSSION THREAD

**Discussion by nsls-j** [hluk/CopyQ#2862] [Q&A]
Hi, 

I have a question to know if i have a bug or a skill issue, i tried several times to change the shortcut to open the main window and nothing changed, it does not make any effect. 

The shortcut is binded on my "ampersand" but i can't see anywhere in my shortcut manager where it is. 

Did I miss something ?

**Comment by hluk** [comment]
Unfortunately, Global shortcuts do not work on Linux in Wayland (I think it is default in recent Ubuntu and other distros).

See also: https://copyq.readthedocs.io/en/latest/faq.html#why-do-global-shortcuts-not-work

**Reply by nsls-j** [reply]
Hi, my appologies because i did not mention that i work on windows.
